templates/config.tfvars.azure-template

```diff
@@ -38,3 +38,12 @@ node_vm_size = "Standard_DS2_v2"
 # Cloudflare proxy (optional)
 # cloudflare_proxied = "true"
 # cloudflare_record_texts = ["record1","record2",...] # Warn: wildcards are not supported when using proxied records
+
+provision = {
+  "action" = {
+    "type"       = "ansible-playbook"
+    "playbook"   = "playbooks/install-core.yml"
+  }
+}
+
+provisioner_image = "provisioner-image-name-autogenerated"
```

Comment:
Add provisioners_img-var to config file (to allow different configs have different version of provisioner image/version tag)